src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Xproj/XprojProjectFactory.cs

```diff
@@ -38,5 +45,15 @@ protected override object PreCreateForOuter(IntPtr outerProjectIUnknown)
             // Should not be called
             throw new NotImplementedException();
         }
+
+        /// <inheritdoc />
+        public Task<IDisposable?> InitializeAsync(ManagedProjectSystemPackage package, IComponentModel componentModel)
+        {
+            IVsProjectFactory factory = new XprojProjectFactory();
```

Comment:
Can this be `this` rather than create a new one?

Reply:
Thanks, fixed. Missed this during conversion from static to instance.